articles/deciding-between-cloud-spanner-and-microsoft-cosmos-db/index.md

```diff
@@ -0,0 +1,55 @@
+
+The globally distributed, multi-model database is available across more than 30 regions worldwide. This is possible considering its turnkey global distribution that automatically scales and replicates data across various data centers in the Azure network.
+
+Cosmos DB&#39;s multi-model feature supports multiple models, including table storage, graphs, key values, and document storage in a single database. It offers high availability and consistency across all these data models, irrespective of their nature, for data storage.
+
+Cosmo DB offers comprehensive [service level agreements](https://azure.microsoft.com/en-us/support/legal/sla/cosmos-db/v1_3/#:~:text=The%20service%20offers%20comprehensive%2099.99,Azure%20regions%2C%20configured%20with%20any) (SLA) of 99.99% for latency, availability, consistency, and throughput. Latency is below 15ms for document write, and 10ms for document read operations. When latency is at a minimum, users experience a faster, seamless experience.
+
+### Pros and Cons of Microsoft Cosmos DB
+
+CosmosDB&#39;s use of data containers projectable as different types of datasets is unique. It provides an SQL interface for documenting projections, including stored procedures and triggers. It is a simple to use service.
+
+Microsoft joined the cloud market and repurposed its on-premises software for the cloud. These software include [Windows Server](https://www.microsoft.com/en-us/windows-server), [SQL Server](https://www.microsoft.com/en-gb/sql-server/sql-server-downloads), [Office](https://www.office.com/), [.Net](https://dotnet.microsoft.com/), [Microsoft Dynamics 365](https://dynamics.microsoft.com/en-us/), and [SharePoint](https://support.microsoft.com/en-us/office/what-is-sharepoint-97b915e6-651b-43b2-827d-fb25777f446f). Many organizations use Windows and other Microsoft associated software, which is essential for Cosmos DB&#39;s success. Cosmos DB tightly integrates with other Microsoft applications. Thus, organizations using a lot of Microsoft software find sense to also use Cosmos DB. This builds on customer loyalty.
+
+On the flip side, users associate Cosmos DB platform with some imperfections. Cosmos DB fails to provide a seamless way to switch between [database-provisioned throughput and container-provisioned throughput](https://docs.microsoft.com/en-us/azure/cosmos-db/set-throughput). You have to recreate the database for you to switch successfully. Besides, Cosmos DB does not allow documents from different logical partition to participate in the same transaction. And writes from different containers cannot be part of the same transaction.
```

Comment:
What does this mean? Database-provisioned throughput and container-provisioned throughput. Could you explain that or attach a link to that. The link you have given does not give information about container provisional throughput.

Reply:
> What does this mean? Database-provisioned throughput and container-provisioned throughput. Could you explain that or attach a link to that. The link you have given does not give information about container provisional throughput.

@lalith1403 Container-provisioned throughput is explained under the link provided. It is under the sub-heading "Set throughput on a container"

Reply:
You may guide the reader to that sub section. The thing is, a reader has limited attention. Including several links leads to dispersion in attention and therefore adds less value.